cmake/dependencies.cmake

```diff
@@ -87,9 +87,3 @@ set(Boost_USE_STATIC_LIBS OFF)
 set(Boost_USE_MULTITHREADED ON)
 set(Boost_USE_STATIC_RUNTIME OFF)
 find_package(Boost 1.53.0 COMPONENTS multiprecision)
```

Comment:
Just tried to compile - failed :( that's because:
1. plz remove version number
2. `COMPONENT multiprecision` -> `REQUIRED` (doesn't find on my platform, but component presents)